### PR TITLE
Move apis under kubeedge/cloud/pkg/apis to kubeedge/pkg/apis

### LINKED ISSUE #3669: Move apis under kubeedge/cloud/pkg/apis to kubeedge/pkg/apis
Let's move all the apis to one place.

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #3669 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/hack/update-codegen.sh [hunk 1]
```diff
@@ -21,6 +21,6 @@ set -o pipefail
 SCRIPT_ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/../.. && pwd)
 
 ${SCRIPT_ROOT}/cloud/hack/generate-groups.sh "deepcopy,client,informer,lister" \
-github.com/kubeedge/kubeedge/cloud/pkg/client github.com/kubeedge/kubeedge/cloud/pkg/apis \
+github.com/kubeedge/kubeedge/cloud/pkg/client github.com/kubeedge/kubeedge/pkg/apis \
 "devices:v1alpha2 reliablesyncs:v1alpha1 rules:v1" \
```
Comment (gy95) [inline cloud/hack/update-codegen.sh:25]: is it Ok to put client and apis to different dir?

Comment (fisherxu) [reply cloud/hack/update-codegen.sh:25]: It's better to move the client to pkg too.

**Comment by gy95** [comment]
report this error, not sure why didn't occur on my local machine, how about deleting this whitenoise check? :)
`Some files have white noise issue, please run `make lint` to solve and `git status` to find and fix this issue`

**Pull request closed without merging**